module/VuFind/src/VuFind/RecordDriver/SolrMarc.php

```diff
@@ -72,4 +75,69 @@ public function getURLs()
             $this->getIlsURLs()
         );
     }
+
+    /**
+     * Takes a Marc field that notes are stored in (ex: 950) and a list of
+     * sub fields (ex: ['a','b']) optionally as well as what indicator
+     * numbers and values to filter for and concatenates the subfields
+     * together and returns the fields back as an array
+     * (ex: ['subA subB subC', 'field2SubA field2SubB'])
+     *
+     * @param string $field    Marc field to search within
+     * @param array  $subfield Sub-fields to return or empty for all
+     * @param array  $indData  Array containing the indicator number as the key
+     *                         and the value as an array of strings for the
+     *                         allowed indicator values
+     *                         ex: [['1' => '1', '2', '2' => '']]
+     *                         would filter ind1 = 1 or 2 and ind2 = blank
+     *
+     * @return array The values within the subfields under the field
+     */
+    public function getMarcFieldWithInd(
+        string $field,
+        ?array $subfield = null,
+        array $indData = []
+    ) {
+        $vals = [];
+        $marc = $this->getMarcReader();
+        $marc_fields = $marc->getFields($field, $subfield);
+        foreach ($marc_fields as $marc_data) {
+            $field_vals = [];
+            // Check if that field has either indicator (MARC only has up to 2 indicators)
+            foreach (range(1, 2) as $indNum) {
+                if (array_key_exists($indNum, $indData)) {
+                    if (in_array(trim(($marc_data['i' . $indNum] ?? '')), $indData[$indNum])) {
+                        $subfields = $marc_data['subfields'];
+                        foreach ($subfields as $subfield) {
+                            $field_vals[] = $subfield['data'];
+                        }
+                    }
+                }
+            }
+            if (!empty($field_vals)) {
+                $vals[] = implode(' ', $field_vals);
+            }
+        }
+        return array_unique($vals);
+    }
+
+    /**
+     * Get the abstract and summary notes
+     *
+     * @return array Note fields from the MARC record
+     */
+    public function getAbstractAndSummaryNotes()
+    {
+        return $this->getMarcFieldWithInd('520', null, [1 => ['', '0', '2', '3', '8']]);
+    }
+
+    /**
+     * Get the location of other archival materials notes
+     *
+     * @return array Note fields from the MARC record
+     */
+    public function getLocationOfArchivalMaterialsNotes()
+    {
+        return $this->getMarcFieldWithInd('544', null, [1 => ['', '0']]);
```

Comment:
Similarly, I think this should be:

```suggestion
        return $this->getMarcFieldWithInd('544', null, [[1 => ['', '0']]]);
```